forc-plugins/forc-publish/src/main.rs

```diff
@@ -29,6 +29,7 @@ async fn main() {
     init_tracing_subscriber(TracingSubscriberOptions::default());
 
     if let Err(err) = run().await {
+        println!();
         println_error(&format!("{err}"));
```

Comment:
note: this may result in empty line to stdout; and error to stderr below; if this is not intended; and you only want stderr output (no stdout) - you may want to:

```suggestion
        println_error(&format!("\n{err}"));
```